kolibri/core/assets/src/views/k-breadcrumbs/index.vue

```diff
@@ -135,6 +143,12 @@
         return Math.min(this.parentWidth - DROPDOWN_SIDE_PADDING, MAX_CRUMB_WIDTH);
       },
     },
+    watch: {
+      items(val) {
+        this.crumbs = Array.from(val);
+        this.attachSensors();
+      },
+    },
```

Comment:
any chance this could cause a memory leak? Looks like we're attaching over and over without ever detaching.

Reply:
@christianmemije What do you think? Based on `attachedSensors` code, I don't see an obvious memory leak, but I'm not sure what the `ResizeSensor` in

```
updatedCrumb.sensor = new ResizeSensor(updatedCrumb.ref, this.throttleUpdateCrumbs);
```

is donig